src/index.html

```diff
@@ -7,12 +7,70 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="header">
+        <img
+          class="header__photo"
+          src="images/imac.jpeg"
+          alt="imac"
+        />
+      </div>
+      <div class="footer">
+        <div class="footer__name">
+          <h1 class="footer__h">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h1>
+        </div>
+        <p class="footer__code">Product code: 195434</p>
+        <div class="footer__rate">
+          <div class="footer__rate-stars footer__rate-stars--4">
+            <div class="footer__rate-stars-star"></div>
+            <div class="footer__rate-stars-star"></div>
+            <div class="footer__rate-stars-star"></div>
+            <div class="footer__rate-stars-star"></div>
+            <div class="footer__rate-stars-star"></div>
+          </div>
```

Comment:
You need to use this block from the previous task
```suggestion
          <div class=stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```